lib/gauth.js

```diff
@@ -38,14 +38,17 @@ async function getAuthClient(support) {
 }
 
 async function getJWTAuthClient() {
-	const content = await fs.readFile(process.env.GAPI_SERVICE_ACCOUNT_JWT);
+	const content = await fs.readFile(
+		process.env.GAPI_SERVICE_ACCOUNT_JWT,
+		'utf8',
+	);
 	const jwt = JSON.parse(content);
 	let auth = new google.auth.JWT({
 		email: jwt.client_email,
 		key: jwt.private_key,
 		scopes: SCOPES,
 	});
-	await auth.authorizeAsync();
+	await auth.authorize();
```

Comment:
The implementation of `auth.authorize` is basically `if (callback) this.authorizeAsync().then(cb) else return this.authorizeAsync()` so this change just adds a bit of indirection but it's necessary to pass the type check